homepage/de.js

```diff
@@ -0,0 +1,67 @@
+{
+	"Integrations": "Integrationen",
+	"Pricing": "Preise",
+	"Apps": "Apps",
+	"Existing user login": "Als bestehender Nutzer anmelden",
+	"login": "anmelden",
+	"Chat, for GitHub.": "Chat, f&uuml;r GitHub.",
+	"Get unlimited public rooms and one-to-one chats, free forever. Private team plans start from $2 per user per month. No credit card required.": "Erhalte unbegrenzte &ouml;ffentliche und private Chatrooms. Bereits ab $2 im Monat pro Benutzer. Keine Kreditkarte notwendig.",
+	"Sign up free with Github": "Kostenlos mit Github registrieren",
+	"Sign up with Github": "Mit Github registrieren",
+	"By signing up you agree to our <a %s>Terms and Conditions</a>": "Mit der Registrierung stimmst du unseren <a %s>Nutzungsbedingungen</a> zu",
+	"Markdown power": "Markdown power",
+	"Ever wanted full markdown support in chat? How about syntax highlighting?": "Wolltest du schon immer volle Markdown Unterst&uuml;tzung in einem Chat? Wie w&auml;rs mit Syntax-Highlighting?",
+	"Yip, we got your back.": "Yip, wir stehen hinter dir.",
+	"Private conversations": "Private Unterhaltungen",
+	"Public rooms are awesome (and free), but we get that some stuff is private. So we've got <a %s>plans for that</a>.": "&ouml;ffentliche Chatrooms sind fantastisch (und kostenlos), aber wir alle haben Dinge die Privat sind. Also haben wir <a %s>Angebote f&uuml;r dies</a>.",
```

Comment:
„für dies“ → „dafür“?

And I believe there should be a comma in „Dinge, die privat sind“ (and „privat“ lowercased).